packages/dappmanager/test/integration/migrations/aliasMigration.test.int.ts

```diff
@@ -0,0 +1,255 @@
+
+const DNP_NAME = "logger.dnp.dappnode.eth";
+const DNP_NAME_PUBLIC = "logger.public.dappnode.eth";
+const DNP_NAME_MONO = "logger-mono.dnp.dappnode.eth"
+const DNP_NAME_MONO_PUBLIC = "logger-mono.public.dappnode.eth";
+
+const TEST_ALIAS_PATH = `dnp_repo/${DNP_NAME}`;
+const TEST_ALIAS_PATH_PUBLIC = `dnp_repo/${DNP_NAME_PUBLIC}`;
+const TEST_ALIAS_PATH_MONO = `dnp_repo/${DNP_NAME_MONO}`;
+const TEST_ALIAS_PATH_MONO_PUBLIC = `dnp_repo/${DNP_NAME_MONO_PUBLIC}`;
+
+
+const DNCORE_NETWORK = params.DNP_PRIVATE_NETWORK_NAME;
+
+const monoContainer: PackageContainer = {
+    ...mockContainer,
+    containerName: "DAppNodeTest-logger.dnp.dappnode.eth",
+    dnpName: `${DNP_NAME_MONO}`,
+    serviceName: `${DNP_NAME_MONO}`,
+    isCore: false,
+    isMain: true,
+};
+
+const monoContainerPublic: PackageContainer = {
+    ...mockContainer,
+    containerName: "DAppNodeTest-logger.public.dappnode.eth",
+    dnpName: `${DNP_NAME_MONO_PUBLIC}`,
+    serviceName: `service`,
+    isCore: false,
+    isMain: true,
+};
+
+const containerMain: PackageContainer = {
+    ...mockContainer,
+    containerName: "DAppNodeTest-logger.main.dnp.dappnode.eth",
+    dnpName: `${DNP_NAME}`,
+    serviceName: "mainService",
+    isMain: true,
+    isCore: false,
+};
+
+const containerNotMain: PackageContainer = {
+    ...mockContainer,
+    containerName: "DAppNodeTest-logger.notmain.dnp.dappnode.eth",
+    dnpName: `${DNP_NAME}`,
+    serviceName: "notmainService",
+    isMain: false,
+    isCore: false,
+};
+
+const containerMainPublic: PackageContainer = {
+    ...mockContainer,
+    containerName: "DAppNodeTest-logger.main.public.dappnode.eth",
+    dnpName: `${DNP_NAME_PUBLIC}`,
+    serviceName: "mainService",
+    isMain: true,
+    isCore: false,
+};
+
+const containerNotMainPublic: PackageContainer = {
+    ...mockContainer,
+    containerName: "DAppNodeTest-logger.notmain.public.dappnode.eth",
+    dnpName: `${DNP_NAME_PUBLIC}`,
+    serviceName: "notmainService",
+    isMain: false,
+    isCore: false,
+};
+
+//We check 6 possible containers: 
+// - monoContainer            --> container comming from a docker-compose with only 1 service. dnp
+// - monoContainerPublic      --> container comming from a docker-compose with only 1 service. public
+// - containerMain            --> container comming from a docker-compose with 2 services, it is maked as the main one. dnp
+// - containerNotMain         --> contaienr comming from a docker-compose with 2 services, it is not marked as the main one. dnp
+// - containerMainPublic      --> container comming from a docker-compose with 2 services, it is maked as the main one. public
+// - containerNotMainPublic   --> contaienr comming from a docker-compose with 2 services, it is not marked as the main one. public
+const containers = [containerMain, containerNotMain, monoContainer, containerMainPublic, containerNotMainPublic, monoContainerPublic];
+
+const CONTAINER_COMPOSE = `
+version: '3.4'
```

Comment:
Here you set version 3.4, but above you migrate to 3.5 right? Why is this? I might be missing something